src/apps/Channels/ChannelsList.jsx

```diff
@@ -77,24 +76,30 @@ export default React.createClass({
       ...other
     } = this.props;
 
-    if ((!items || !items.length) && !isLoading) {
-      return (
-        <EmptyView
-          data-e2e="real-time-channels-empty-item"
-          iconClassName="synicon-socket-channel"
-          iconColor={Colors.blue500}
-          title="Real-time Channel"
-          urlLabel="Real-time Channel"
-          description="Create Channel which is a way of providing realtime communication functionality."
-          docsUrl="http://docs.syncano.io/docs/realtime-communication"
-          handleClick={Actions.showDialog}
-          buttonLabel="Add Real-time Channel"
-        />
-      );
-    }
+    // Now there is always a default channel created at start
+    // https://github.com/Syncano/syncano-platform/pull/1407
```

Comment:
that's because of backend change. we have to decide if we want to display or hide it, but for me it's an separate issue.